x-pack/.gitignore

```diff
@@ -12,6 +12,3 @@
 !/legacy/plugins/infra/**/target
 .cache
 !/legacy/plugins/siem/**/target
-
-# We don't want any yarn.lock files in here
-/yarn.lock
```

Comment:
Now that we remove this part, how can we make sure that actual dependencies are not added to the new lock file, but that it's only used as a symlink? Will yarn automatically discover that it's a symlink-style lock file and not also start adding actual dependencies to it?

Reply:
This line in .gitignore was actually unnecessary, because this package is part of a Yarn Workspace. The only reason I removed it was so I could add the symlink here, like I've added it elsewhere.

So, because this package is part of a Yarn Workspace, Yarn knows where the lockfile is for this package is; the adjacent symlink is purely cosmetic.